### PR TITLE
Fixes #31393 - Drop sysvinit scripts on Debian

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Review by lzap — APPROVED** [review]
Looks okay to me. It built so... let's ship it!

File: debian/buster/foreman-proxy/foreman-proxy.preinst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+dpkg-maintscript-helper rm_conffile /etc/init.d/foreman-proxy 2.4.0~ -- "$@"
+dpkg-maintscript-helper rm_conffile /etc/default/foreman-proxy 2.4.0~ -- "$@"
```
Comment (evgeni) [inline debian/buster/foreman-proxy/foreman-proxy.preinst:3]:
```suggestion
dpkg-maintscript-helper rm_conffile /etc/default/foreman-proxy 2.4.0~ -- "$@"

#DEBHELPER#
```

File: debian/bionic/foreman-proxy/foreman-proxy.preinst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+dpkg-maintscript-helper rm_conffile /etc/init.d/foreman-proxy 2.4.0~ -- "$@"
+dpkg-maintscript-helper rm_conffile /etc/default/foreman-proxy 2.4.0~ -- "$@"
```
Comment (evgeni) [inline debian/bionic/foreman-proxy/foreman-proxy.preinst:3]:
```suggestion
dpkg-maintscript-helper rm_conffile /etc/default/foreman-proxy 2.4.0~ -- "$@"

#DEBHELPER#
```

**Review by evgeni — CHANGES_REQUESTED** [review]

**Comment by ekohl** [comment]
Updated

**Review by evgeni — APPROVED** [review]

**Pull request merged**